ai_genomics/utils/reading.py

```diff
@@ -9,3 +11,13 @@ def read_json(data: Union[pathlib.Path, str]) -> List[Dict]:
     """
     with open(data) as json_file:
         return json.load(json_file)
+
+
+def fetch_s3(s3_path) -> Union[pd.DataFrame, Dict]:
```

Comment:
we have a function that does this also in ai_genomics.getters.data_getters (load_s3_data) (and indeed similar script of loading and saving data)- you may just want to use that one instead